### PR TITLE
MAT-8045: update translator and cqframework

### PR DESCRIPTION
**Pull request by etan-sb**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-8045](https://jira.cms.gov/browse/MAT-8045)
(Optional) Related Tickets:

### Summary

`madie-translator-commons` was updated as part of https://github.com/MeasureAuthoringTool/madie-translator-commons/pull/39

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ madie:
       cql:
         uri: /cql-libraries/cql
   translatorVersion:
-      currentVersion: ${CURRENT_TRANSLATOR_VERSION:3.14.0}
-      mostRecentVersion: ${MOST_RECENT_TRANSLATOR_VERSION:3.14.0}
+      currentVersion: ${CURRENT_TRANSLATOR_VERSION:3.24.0}
+      mostRecentVersion: ${MOST_RECENT_TRANSLATOR_VERSION:3.24.0}
```
Comment (adongare) [inline src/main/resources/application.yaml:33]: should this be same as well https://github.com/MeasureAuthoringTool/madie-fhir-elm-translator/pull/56/files?

**Review by nmorasb — APPROVED** [review]
